try {
    return x % y;
    return a + b;
    alert('Action completed!');
} catch (error) {
    console.error("Error:", error);
}